api/src/main/java/com/netflix/iceberg/Tables.java

```diff
@@ -16,12 +16,8 @@
 
 package com.netflix.iceberg;
 
-class Tables {
-  public static Table create(String path, Schema schema, PartitionSpec spec) {
-    return null;
-  }
+public interface Tables {
+  Table create(Schema schema, PartitionSpec spec, String location);
```

Comment:
Is there a better name than `location` for this since it may be a db/table or catalog/db/table?